<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>modifch</class>
 <widget class="QDialog" name="modifch">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>996</width>
    <height>623</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>-10</y>
     <width>171</width>
     <height>731</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(11, 22, 44);</string>
   </property>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>50</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe Print</family>
      <pointsize>11</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 10px;
background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Retour</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="widget_2" native="true">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>10</y>
     <width>671</width>
     <height>211</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(235, 172, 162);
border-radius: 40px;</string>
   </property>
   <widget class="QLineEdit" name="lineEdit_ID">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>60</y>
      <width>271</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
border-radius: 10px;
</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>20</y>
      <width>311</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe Print</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Veuillez entrer le CIN</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="widget_3" native="true">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>240</y>
     <width>651</width>
     <height>321</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(235, 172, 162);
border-radius: 40px;</string>
   </property>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>201</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe Print</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Nouvelle Adresse</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_nouveauNumeroTelephone">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>40</y>
      <width>191</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
border-radius: 10px;
</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>191</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe Print</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Nouveau  numtel</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_nouvelleAdresse">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>80</y>
      <width>191</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
border-radius: 10px;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>241</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe Print</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Nouveau email</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_nouvelEmail">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>130</y>
      <width>191</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
border-radius: 10px;</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>570</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe Print</family>
     <pointsize>11</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Enregistrer</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
